packages/block-editor/src/components/inserter/quick-inserter.js

```diff
@@ -46,7 +51,11 @@ function QuickInserterList( {
 	onHover,
 } ) {
 	const shownBlockTypes = useMemo(
-		() => blockTypes.slice( 0, SHOWN_BLOCK_TYPES ),
+		() =>
+			orderBy( blockTypes, [ 'frecency' ], [ 'desc' ] ).slice(
```

Comment:
Should we implement this inside `searchBlockItems`? It seems business logic related to the search function. What do you think?

Reply:
No, I don't think so, in fact, it was initially in the selector (so applied to everything) but due to design decisions that are per component, we moved it out, and now there are places where we want things to be sorted and others not.

Reply:
Yeah, I meant as a new param in searchBlockItems.

Reply:
It's not always tied to search. For instance if the search input is empty, this list is rendered still without calling `searchBlockItems`